src/Symfony/Component/DomCrawler/Crawler.php

```diff
@@ -866,6 +866,136 @@ public static function xpathLiteral($s)
     }
 
     /**
+     * @deprecated Using the SplObjectStorage API on the Crawler is deprecated as of 2.8 and will be removed in 3.0.
+     */
+    public function attach($object, $data = null)
+    {
+        @trigger_error('The method '.__METHOD__.' is deprecated as of 2.8 and will be removed in 3.0.', E_USER_DEPRECATED);
+
+        parent::attach($object, $data);
+    }
+
+    /**
+     * @deprecated Using the SplObjectStorage API on the Crawler is deprecated as of 2.8 and will be removed in 3.0.
+     */
+    public function detach($object)
+    {
+        @trigger_error('The method '.__METHOD__.' is deprecated as of 2.8 and will be removed in 3.0.', E_USER_DEPRECATED);
```

Comment:
We are using `The __METHOD__ method` in triggers. Should be changed everywhere here.